crates/router/src/connector.rs

```diff
@@ -16,9 +16,12 @@ pub mod utils;
 pub mod worldline;
 pub mod worldpay;
 
+pub mod opayo;
+
 pub use self::{
     aci::Aci, adyen::Adyen, applepay::Applepay, authorizedotnet::Authorizedotnet,
     braintree::Braintree, checkout::Checkout, cybersource::Cybersource, fiserv::Fiserv,
     globalpay::Globalpay, klarna::Klarna, payu::Payu, rapyd::Rapyd, shift4::Shift4, stripe::Stripe,
     worldline::Worldline, worldpay::Worldpay,
+opayo::Opayo,
```

Comment:
run cargo clippy --all-features --all-targets && cargo +nightly fmt --all and fix clippy warnings